### PR TITLE
Wip fix travis build

### PR DESCRIPTION
**Pull request by benrs**
Ignore the amount of commits, a lot of it was just playing around with the travis.yml. I changed the way we output the unicode chess and box characters. I assume that the user has the proper encoding.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,22 +2,17 @@ dist: trusty
...
-      - clang-3.7
-      - g++-5
-      - gcc-5
-script:
-  # Check versions of gcc, g++ and cmake
-  - gcc -v && g++ -v && cmake --version
-  - mkdir build && cd build
-  - cmake ../
-  - make
-  - ./stocklion_tests
```
Comment (jacobrs) [inline .travis.yml]: you still need to run the tests.

File: datastructures/Board.cpp [hunk 2]
```diff
@@ -156,7 +156,7 @@ void Board::printUnicodeBoard() {
             if(i == 0)
                 std::cout << (char)('A' + j - 1) << "   ";
             else {
-                char32_t token = ' ';
+                std::string token = " ";
```
Comment (jacobrs) [inline datastructures/Board.cpp:159]: can this be a char? or is it a string because the new unicode values are strings?

**Review by jacobrs — COMMENTED** [review]
Looks good but you need to run the tests in the travis job. Also there are merge conflicts with master.

File: datastructures/Board.cpp (hunk 2 above)
Comment (benrs) [reply datastructures/Board.cpp:159]: It can't be a char sadly, doesn't fit. Unicode chars vary in size.

**Review by jacobrs — APPROVED** [review]
travis sucks with c++ but it works now.

**Pull request merged**